### DISCUSSION TITLE
【委托】巨额报酬等你来拿（传奇绳匠优先）

### DISCUSSION THREAD

**Discussion by ab1efe** [share121/inter-knot#461] [其他]
![HTI)@V3GV ZEA8NBDLY}{(R](https://github.com/user-attachments/assets/6df7da75-c4c2-4717-88e8-5875bc6ac072)
据我大姨的二姑的爷爷的儿子的表妹的可靠消息，空洞中时常会出现说人话的邦布，这种邦布在空洞中利用裂隙来去自如非常难以捕捉，猜测是因为空洞侵蚀而导致的变异，其稀有程度绝不亚于白金邦布，不！甚至可能更高！作为一个骨灰级邦布收藏家岂能错过这种机会！抓住稀有邦布带到我面前的人除了基础的1000万丁尼报酬外还将额外附赠那个超有名的大企业远景实业公司所建造的二层别墅一栋。
有本事想赚钱的绳匠们行动起来，先到先得！

**Comment by yiwangzj** [comment]
整个绳网上都知道传奇绳匠法厄同就是会说话的邦布，你的曾姨姥姥消息来源不够可靠啊。

**Comment by lllliufeng** [comment]
那只邦布摇人怎么办？

**Comment by swyh51982** [comment]
这个委托我接了

**Comment by ProxyPhaeton** [comment]
别想打我家邦布的主意！

**Comment by XiaoFffeng** [comment]
朋友，我奉劝你不要打这只邦布的主意。不然它之后就会摇来一个搓火刀的家伙对着你玛卡巴卡

**Comment by Mnpolize** [comment]
谁家可爱伊埃斯，抱走了

**Reply by ProxyPhaeton** [reply]
我是法厄同😡

**Reply by Mnpolize** [reply]
让姨姨亲亲你家伊埃斯好不好😘😘😘😘😘😘😘

**Reply by ProxyPhaeton** [reply]
珂蕾妲找你，格莉丝

**Comment by Luxe6** [comment]
这个委托我接了！

**Comment by 1155993355** [comment]
把伊埃斯还给我，那是我法厄同的

**Reply by ProxyPhaeton** [reply]
看着我的ID说话！😡

**Comment by kingquagen** [comment]
接不了。

**Comment by biliqide** [comment]
妮可老大说了，只要价格够高，10个也能搞来

**Comment by 1155993355** [comment]
比利，或许我们该谈谈我们之间的债务了

**Comment by BEILUO126** [comment]
这是新艾利都最强的杀手邦布，你敢和他对视十秒吗？

**Comment by Hongxuli2006** [comment]
委托接了
